lib/env/cli.rb

```diff
@@ -145,8 +145,16 @@ def cli_syntax(command, args_str = nil)
     command :'set-default' do |c|
       cli_syntax(c, '[NAME]')
       c.description = "Set the default application environment"
-      c.option '--remove', 'Remove the default application environment if it is set.'
```

Comment:
We're likely to have a number of users how are used to using `set-default --remove`.  With this change that will now error without providing any help to them to discover the new `remove-default` command.

Can you add this option back and have it print to stderr that this option is deprecated and that the `remove-default` command should be used instead.  Ideally, we'd also call the `RemoveDefault` command if easy to implement.